src/server/controllers/memberController.js

```diff
@@ -6,6 +6,7 @@
  * And we resend requests account verification emails.
  */
 
+const locale = require('date-fns/locale')
```

Comment:
Can you merge this with the line beneath? Those imorts are from the same package.

Reply:
deconstructing doesn't work i have to work the / syntax

Reply:
probably because the package doesn't export local so its the only way to access locale

Reply:
oh its because its a sub-module thats why